internal/cli/watch_opts.go

```diff
@@ -55,6 +67,27 @@ func (opts *WatchOpts) Watch(f Watcher) error {
 	}
 }
 
+func (opts *WatchOpts) exponentialBackoff(err error) bool {
+	if opts.n <= 8 && checkForError(err, "CLUSTER_NOT_FOUND") {
+		backoff := math.Pow(base, float64(opts.n))
```

Comment:
if I'm following right your backoff logic is 4 seconds, 16 seconds and finally 256 seconds. That look like a lot.

How about 1 second, 2 seconds and lastly 4 seconds?

Reply:
You're right, those waiting time may be exaggeration. I will update it to a much smaller one, as you're suggesting.